trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './base_decorator'
+require_relative './person'
+require_relative './capitalize_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    return @nameable.correct_name[0, 10] if @nameable.correct_name.length > 10

```

Comment:
- [ ] if you want the name to have a max length of 10 characters it should look like `return @nameable.correct_name[0, 9] if @nameable.correct_name.length > 10`